metricbeat/mb/registry.go

```diff
@@ -172,14 +243,14 @@ func (r *Register) Modules() []string {
 	return modules
 }
 
-//MetricSets returns the list of metricsets registered for a given module
+// MetricSets returns the list of MetricSets registered for a given module
 func (r *Register) MetricSets(module string) []string {
 	r.lock.RLock()
 	defer r.lock.RUnlock()
 
 	var metricsets []string
 
-	sets, ok := r.metricSets[module]
+	sets, ok := r.metricSets[strings.ToLower(module)]
```

Comment:
I wonder if we should add a test somewhere to make sure this is not missing in other parts of the code.